src/services/legacyStats.ts

```diff
@@ -0,0 +1,26 @@
+import { UserContactEvent, UserStatsState } from '../redux/userStats';
+
+export interface LegacyUserStatsState {
+  all: UserContactEvent[];
+  unavailable: number;
+  voicemail: number;
+  contact: number;
+}
+
+export const transform = (legacyStats: LegacyUserStatsState): UserStatsState => {
+  let userStats: UserStatsState = {
+    all: legacyStats.all.map(i => {
+      return {
+        contactid: i.contactid,
+        issueid: i.issueid,
+        result: i.result,
+        time: i.time,
+      };
+    }),
+    unavailable: legacyStats.unavailable,
```

Comment:
We were not very consistent in choo. These result counts are actually named `contacted`, `vm` and `unavailable` in choo's userstats. This would be the last place we need to use these old (and inconsistent) names.

Reply:
OK. I wasn't sure.  I have both(contacted/contact, voicemail/vm) in my current choo user stats.  Maybe this was from a while ago.

I've changed it to contacted/vm/unavailable.